backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/process/ProcessExecutionResult.java

```diff
@@ -67,6 +71,9 @@
 @JsonAutoDetect(fieldVisibility = Visibility.ANY, getterVisibility = Visibility.NONE, setterVisibility = Visibility.NONE)
 public class ProcessExecutionResult
 {
+	private static String DEBOUNCER_BUFFER_MAX_SIZE_SYSCONFIG_NAME = "de.metas.process.debouncer.bufferMaxSize";
```

Comment:
- please make the names of the sysconfigs clearer. e.g. `de.metas.process.pinstaceLogPersister.debouncer...`
- also, shold be final